src/components/MetaData.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
```

Comment:
We should give this component a more general name, like `Pill.svelte`. The idea is that we could use it in other places as well.